### ISSUE TITLE
Refresh error

### ISSUE THREAD

**Issue by bolmstedt** [lanayotech/vagrant-manager#107]
I'm getting errors on refresh, saying there was an error running vagrant global-status --prune. When I run it myself, it doesn't fail:
```
id       name    provider   state    directory
-------------------------------------------------------------------------
3d15bc7  default virtualbox poweroff /Users/marx/.vagrantboxes/xxx
7186d66  default virtualbox poweroff /Users/marx/.vagrantboxes/yyy
72a7577  default virtualbox running  /Users/marx/.vagrantboxes/zzz
906cb78  default virtualbox poweroff /Users/marx/.vagrantboxes/xyz

The above shows information about all known Vagrant environments
on this machine. This data is cached and may not be completely
up-to-date. To interact with any of the machines, you can go to
that directory and run Vagrant, or you can use the ID directly
with Vagrant commands from any directory. For example:
"vagrant destroy 1a2b3c4d"
```

Looked at the source and tried:
```bash
/bin/bash -l -c vagrant global-status --prune 2> /dev/null
```
This fails, because quotation marks are missing around the command.
```bash
/bin/bash -l -c "vagrant global-status --prune 2> /dev/null"
```
The above works.

Don't know if that is how that task thin in the VagrantRefreshStatusScanner works, but it's they only reason I can figure out with my limited knowledge.

Vagrant 1.7.4
Vagrant Manager 2.4.4
OSX 10.10.3

**Comment by dorongutman** [comment]
Happens to me as well - the global status command returns a valid response but Vagrant manager says it's not.

Vagrant 1.7.4
Vagrant Manager 2.4.4
OSX 10.10.4

**Comment by amitailanciano** [comment]
I'll put together a debug build for you guys to test with, I'll try to post it here by this evening

**Comment by amitailanciano** [comment]
Unrelated, but see #111 for a temporary resolution

**Comment by fuzzy76** [comment]
Seeing this on Vagrant 1.7.4, Vagrant Manager 2.4.4 and OS X 10.11. There is no solution in #111 as far as I can tell?

**Comment by chriswayoub** [comment]
This should be resolved in 2.5.0. If you are still having this issue, please re-open the ticket.
